src/spatial/views/source_metadata_view.py

```
@@ -0,0 +1,126 @@
+
+        try:
+            data = SourceMetadataModel().build_source_metadata(
+                latitude=lat,
+                longitude=lon,
+                start_date=start_date,
+                end_date=end_date,
+                pollutants=pollutants,
+                include_satellite=include_satellite,
+            )
+            return jsonify({"message": "Operation successful", "data": data}), 200
+        except Exception as ex:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)